lib/ngtcp2_crypto.c

```diff
@@ -86,6 +86,13 @@ static size_t varint_paramlen(uint64_t param) {
   return 4 + ngtcp2_put_varint_len(param);
 }
 
+static uint8_t *put_uint16_varint(uint8_t *p, ngtcp2_transport_param_id id,
+                                  uint64_t value) {
+  p = ngtcp2_put_uint16be(p, id);
```

Comment:
It looks like we need to cast id to (uint16_t).

Reply:
Sorry about that, I ran out of time on Friday. I'll update shortly, thanks!